srcd/superset/assets/stylesheets/less/custom-brand.less

```diff
@@ -1,134 +1,10 @@
-/**
```

Comment:
Why some of the variables defined here have not been moved?

Reply:
@se7entyse7en do you mean: "moved to [`./variables-override.less`](https://github.com/src-d/sourced-ui/pull/226/files#diff-14ccc45e35ab92af261d436268dd9b21)"?

If so, I only moved to `./variables-override.less`, those that:
- do not exist in [`./cosmo/variables.less`](https://github.com/src-d/sourced-ui/pull/226/files#diff-a25722b6f21ec29571236aee3020385c)
- do override a variable from `./cosmo/variables.less`

Why? because since we're already importing `./cosmo/variables.less`, it's not needed to override it with the very same value &rarr; it would not be clear which ones are kept equal, and which ones are overridden.

Reply:
Thanks!